### DISCUSSION TITLE
接入微信后，ai回复（我现在有点累了，等会再来吧）

### DISCUSSION THREAD

**Discussion by 1452534914** [zhayujie/chatgpt-on-wechat#1033] [Q&A]
/
![mmexport1683318514548](https://user-images.githubusercontent.com/119047908/236611973-a22acc88-5efb-4f9e-951a-d2a795cb348e.jpg)

**Comment by DPPCoder** [comment]
换个key,免费额度用完了

**Reply by jinshi6** [reply]
可是这是我第一次使用啊，就用完了？
